README.md

```diff
@@ -130,6 +130,138 @@ Number of entries returned 1
 ----------------------------
 ```
 
+## User Agreements
+
+User agreements are handled by a new object type ``fasagreement``.
+Agreements can be linked to 0..n groups. Users are able to consent to
+any enabled user agreement. They can neither consent to disabled user
+agreement nor retract consent. User agreements can be managed by
```

Comment:
If we ever need to allow users to retract consent in the future (possibly removing them from linked groups), would it be complex to do with the way it's implemented now? Would it break things?

Reply:
At the moment I block write operations when a user is a member of an agreement. It's possible to change that later, perhaps with a flag that allows self-removal.

Reply:
My first reply didn't answer your question. The current design does neither make it more complicated nor easier to change the behavior in the future. It's merely ``s/add/remove/`` on the deny rule.